backend/graphql/resolvers/testSessionResolvers.ts

```diff
@@ -68,7 +69,7 @@ const testSessionResolvers = {
       if (!results) return [];
 
       const { students } = await classService.getClassById(parentClass);
-      const resultsByStudentId = results.reduce<
+      const resultsByStudentId = computePercentiles(results, "score").reduce<
```

Comment:
do we re-compute percentile every time we query for student results?

Reply:
Yes, I had remembered that we were deferring optimization until later. It's just a sort so I'm not too too worried about performance, but ultimately we should probably move it to the completion mutation or something. Do you think we should do that now?

Reply:
all good. approving!